test-packages/e2e-tests/test/generator-cli.spec.ts

```diff
@@ -60,4 +60,23 @@ describe('generator-cli', () => {
     expect(entities).toContain('TestEntity.ts');
     expect(entities).toContain('package.json');
   }, 60000);
+
+  it('should set version when versionInPackageJson parameter is used', async () => {
```

Comment:
[req] parameter => option
```suggestion
  it('should set version when versionInPackageJson option is used', async () => {
```